Update Feb 20 PCM notes

TomSweeneyRedHat opened the pull request:
I had trouble getting pages to resolve when trying to post this in February.  So I ended up posting a rough draft.  Florent Benoit has since fixed the problem, so here is the polished off version of the Feb 2024 Cabal notes.

TomSweeneyRedHat commented:
@rhatdan @kedark3 PTAL
@rhatdan, please do NOT merge this until Monday.

kedark3 commented:
Thanks @TomSweeneyRedHat

rhatdan commented:
LGTM

kedark3 commented:
Thanks @rhatdan @TomSweeneyRedHat  - does this take effect same day to update podman.io or takes some time/manual intervention? just curious.

TomSweeneyRedHat commented:
@kedark3 once the merge button is pressed, it all gets published automatically and posted on the site.  Usually it takes 5 to 10 minutes to complete, sometimes a bit longer.  I'm a little concerned about this one though, I would have expected it to be up by now....

kedark3 commented:
@TomSweeneyRedHat I've been getting a lot of that lately. Anything I touch becomes a `special case` lol :laughing:

TomSweeneyRedHat committed:
Update Feb 20 PCM notes

I had trouble getting pages to resolve when trying to post this
in February.  So I ended up posting a rough draft.  Florent Benoit
has since fixed the problem, so here is the polished off version
of the Feb 2024 Cabal notes.

I've also added the March 2024 Cabal notes.

Signed-off-by: tomsweeneyredhat <tsweeney@redhat.com>

TomSweeneyRedHat commented:
Going to smash this through.
